addons/hr/static/tests/web/m2x_avatar_user.test.js

```diff
@@ -71,7 +58,7 @@ test("avatar card preview with hr", async () => {
         </kanban>`,
     });
     await contains(".o_m2o_avatar > img").click();
-    expect.verifySteps(["user read"]);
+    await animationFrame();
```

Comment:
Remove arbitrary timing, use `contains` from mail helpers instead (or `waitFor` from web).
`expect` is really not fit for DOM changes.